hack/update-codegen.sh

```diff
@@ -280,7 +280,6 @@ ensure-in-gopath
 old=${GOCACHE:-}
 export GOCACHE=$(mktemp -d -t codegen.gocache.XXXX)
 export GO111MODULE=on
-export GOPROXY=https://proxy.golang.org
 export GOSUMDB=sum.golang.org
```

Comment:
👍  on removing this

Although not related to your change, why do we export GOSUMDB to the default value?

Reply:
ref: https://golang.org/ref/mod#authenticating

>The default value of GOSUMDB is sum.golang.org, the public checksum database operated by Google

Reply:
I've no idea :grin:  I guess it's reasonable enough to provide a very consistent environment when doing the codegen, but it feels probably overly defensive to force the default gosumdb.